yb-voyager/src/utils/utils.go

```diff
@@ -427,10 +428,7 @@ func GetEnvAsInt(key string, fallback int) int {
 }
 
 func GetMapKeysSorted(m map[string]*string) []string {
-	keys := make([]string, 0, len(m))
-	for key := range m {
-		keys = append(keys, key)
-	}
+	keys := lo.Keys[string, *string](m)
```

Comment:
Do you really need to pass the type parameters `[string, *string]`? The types should have been inferred by the compiler.